manifests/g/GHannesDE/FoxisWorld/1.1/GHannesDE.FoxisWorld.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: GHannesDE.FoxisWorld
+PackageVersion: 1.1
+PackageLocale: en-US
+Publisher: GHannesDE
+PackageName: FoxisWorld
+License: none
+ShortDescription: FoxisWorld Setup

```

```suggestion
ShortDescription: FoxisWorld Setup
```
Since you're the developer, please write a better description. I don't know what this application is, and it's not showing up for me on Google Search.